tests/e2e_web3js_test.go

```diff
@@ -37,40 +42,48 @@ func TestWeb3_E2E(t *testing.T) {
 	})
 
 	t.Run("batch run transactions", func(t *testing.T) {
-		runWeb3TestWithSetup(t, "eth_batch_retrieval_test", func(emu emulator.Emulator) error {
-			tx1, err := cadence.NewString("f9015880808301e8488080b901086060604052341561000f57600080fd5b60eb8061001d6000396000f300606060405260043610603f576000357c0100000000000000000000000000000000000000000000000000000000900463ffffffff168063c6888fa1146044575b600080fd5b3415604e57600080fd5b606260048080359060200190919050506078565b6040518082815260200191505060405180910390f35b60007f24abdb5865df5079dcc5ac590ff6f01d5c16edbc5fab4e195d9febd1114503da600783026040518082815260200191505060405180910390a16007820290509190505600a165627a7a7230582040383f19d9f65246752244189b02f56e8d0980ed44e7a56c0b200458caad20bb002982052fa09c05a7389284dc02b356ec7dee8a023c5efd3a9d844fa3c481882684b0640866a057e96d0a71a857ed509bb2b7333e78b2408574b8cc7f51238f25c58812662653")
```

Comment:
I know that it's impossible to understand what these 2 transactions did :sweat_smile: 
I believe it was a good test case scenario though, it was basically a contract deployment, followed by a contract function call. But I suppose that such logic must already have been tested against in the `flow-go` repository.

Reply:
I plan to add more complex transactions as well, but I prefer actually building them not just passing payload, that way is clear what they do and it's  easier to change.